include/types.h

```diff
@@ -128,7 +128,7 @@ enum NETPLAN_ERROR_DOMAINS {
 /**
  * @brief   Errors for domain @ref NETPLAN_PARSER_ERROR.
  * @details Such errors are expected to contain the file name,
- *          line and column numbers.
+ *          line and coulumn numbers.
```

Comment:
typo: I think the original spelling was correct here.

Reply:
Of course. Thank you.